README.md

```diff
@@ -4,9 +4,151 @@ Publisher: GreyNoise \
 Connector Version: 2.3.3 \
 Product Vendor: GreyNoise \
 Product Name: GreyNoise \
-Minimum Product Version: 5.5.0
-
-This app provides investigative capabilities using the GreyNoise plugin
+Minimum Product Version: 6.4.1
+
+This app provides investigative capabilities using the GreyNoise plugin and supports receiving alerts and feeds via webhook from GreyNoise
+
+## ⚠️ Playbook Backward Compatibility
+
+- GreyNoise SDK version is upgraded to v3.0.1 in connector version 3.0.0.
+- With this version, There are changes in:
+
+1. Data path for various actions.
+1. CEF fields ingested via on-poll action.
+1. Functionality of **community lookup ip** and **riot lookup ip** can be achived via **ip reputation** action.
```

Comment:
This should indicate the functions are removed in addition to stating the ip rep command replaces them.

Reply:
It’s a bit odd, changes were pushed as part of this [commit](https://github.com/GreyNoise-Intelligence/greynoise-splunk-soar/pull/1/commits/6b980a77dc62489577eed704d6cf68fef58086f3).

I also pushed some minor updates afterward. Could you please confirm if you can see these changes on your end?